### PR TITLE
1.x.x initial

### PR DESCRIPTION
**Pull request by Crazylord**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Draft Theme 1.0.0, 2017-05-07
```
Comment (T2L) [inline CHANGELOG.md]: Should be 
```
## Draft Theme 1.x.x
```

File: draft_theme.info.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Draft theme
+type: theme
```
Comment (T2L) [inline draft_theme.info.yml:2]: Add `hidden: true`

File: draft_theme.theme [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?php
```
Comment (T2L) [inline draft_theme.theme]: Remove empty file

**Review by T2L — COMMENTED** [review]
Remove favicon and screenshot

File: draft_theme.info.yml (hunk 2 above)
Comment (Crazylord) [reply draft_theme.info.yml:2]: Done

**Pull request merged**